### ISSUE TITLE
Version numbers not updating now

### ISSUE THREAD

**Issue by ylluminate** [dmarmor/epichrome#283]
Hey @epichrome, finally getting around to updating things... :) Noticed that version numbers are not updating from 2.3.0b* to the new version numbers that are shown in Finder.

**Comment by dmarmor** [comment]
I'm not sure I understand the issue. Can you explain in a bit more detail? If the later version numbers are shown in Finder, where are you seeing the old 2.3.0b* version numbers?

**Comment by ylluminate** [comment]
Sure, here's an example from Path Finder:
![image](https://p153.p0.n0.cdn.getcloudapp.com/items/7Kubgdd2/example_in_finder.jpg)

Info:
![image](https://p153.p0.n0.cdn.getcloudapp.com/items/JruqzNWe/image.jpg)

**Comment by dmarmor** [comment]
Sorry, still confused. So in Finder, these apps all show as being some later version, but in Pathfinder they show the old version? I've never used Pathfinder so I'm not sure what this represents. Is it possible it's just cached the old version number somehow?

The best test would be to spot check an app or two by looking at their Info.plist file. If it has the current (or expected) version, then the app has correctly updated, and probably Pathfinder is showing old info.

**Comment by ylluminate** [comment]
So initially what threw me off was that I looked into a plist for what I thought was an updated Epichrome app - apparently it was my mistake because I have so many that I must have looked at one that did not succeed in updating.  This was then pronounced by Path Finder not auto-refreshing the version info from the plists and cascaded into my thinking NONE of the plist version numbers were updated. It was a lengthy process fraught with failure on any version prior to 2.3 so I was very frustrated when this happened.  I see now that it was just a perfect storm for me and the version info *IS* updated for the apps that did successfully update.  Thanks!
